constants/general.constant.js

```diff
@@ -11,17 +11,20 @@ const HACKER_STATUS_NONE = "None";
 const HACKER_STATUS_APPLIED = "Applied";
 const HACKER_STATUS_ACCEPTED = "Accepted";
 const HACKER_STATUS_WAITLISTED = "Waitlisted";
+const HACKER_STATUS_DECLINED = "Declined";
+
```

Comment:
nit: what's this break for? if separating stages between applied --> status --> status, seems like everything after this break is applied --> accepted --> status so there would be 3 breaks if it follows that logic? idk maybe a comment above each to explain that